src/AWSSQS.jl

```diff
@@ -325,17 +287,7 @@ end
 Approximate number of messages not visible in a queue.
 """
 function sqs_busy_count(queue::AWSQueue)
-
-    parse(Int,sqs_get_queue_attributes(queue)["ApproximateNumberOfMessagesNotVisible"])
+    return parse(Int,sqs_get_queue_attributes(queue)["ApproximateNumberOfMessagesNotVisible"])
```

Comment:
If you add `return` here you might as well add it every similar place (e.g., `sqs_count`). Also, should be a space after the comma.